### PR TITLE
Announce opam 2.3.0~rc1

### PR DESCRIPTION
**Pull request by kit-ty-kate**
A slightly boring (no change) release ^^

### CODE REVIEW

File: opam-2-3-0-rc1.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "David Allsopp - Tarides" {"mailto:david(à)tarides.com"}
+]
+date: "2024-10-30"
+--BODY--
+
+_Feedback on this post is welcomed on [Discuss](https://discuss.ocaml.org/t/ann-opam-2-3-0-rc1/XXXXX)!_
+
+We're happy to announce the first and hopefully only release candidate of opam 2.3.0.
+
+This version does not have any significant change compared to the previous 2.3.0~beta2 release
```
Comment (rjbou) [inline opam-2-3-0-rc1.md:14]: The is no change in opam itself, it's the release script that changed. The new binary itself was already distributed with last release.

Comment (kit-ty-kate) [reply opam-2-3-0-rc1.md:14]: but that was already announced in the previous blog post (because it was distributed with the last release indeed)

Comment (rjbou) [reply opam-2-3-0-rc1.md:14]: yes

Comment (rjbou) [reply opam-2-3-0-rc1.md:14]:
```suggestion
This version does not have any change compared to the previous 2.3.0~beta2 release
```

Comment (kit-ty-kate) [reply opam-2-3-0-rc1.md:14]: mmh, i'm not sure. That seems a bit wrong given at least the version number changes. It's still a change but not significant

Comment (rjbou) [reply opam-2-3-0-rc1.md:14]: Hmmm... I see.

**Review by rjbou — APPROVED** [review]

**Pull request merged**